SADL/AM-CDM-base.sadl

```diff
@@ -129,3 +129,29 @@ DatasetMetadata (note "Metadata associated with a dataset.") is a class,
 
 DatasetAccessControl (note "Possible types of data access control.") is a class, must be one of {Restricted, Consent, Public}.
+
+Project (note "A temporary endeavor undertaken to create one or more unique AM products or AM services. Links to all other information; may include platform, criticality, sponsors, material DB ID, organization, distribution control (e.g., ITAR restricted or not).") is a class,
+	described by projectID (note "Reference to the associated Project that produced this build. Links to Project record.") with a single value of type string,
+	described by projectCreator (note "Organization that created the project.") with a single value of type Organization,
+	described by projectSponsor (note "Provider of money and requirements for the project.") with a single value of type Organization,
+	described by projectOwningOrganization (note "Organization that owns the project.") with a single value of type Organization,
+	described by projectDistributionStatement (note "Documentation of any restrictions of access or distribution of the data collected and analyzed under this project, by organizations other than the authoritative source.") with a single value of type string,
+	//described by projectDistributionControl (note "ITAR-like restrictions") with a single value of type string,
+	described by projectRequirements (note "Documented requirements for the project.") with a single value of type Document,
+	described by projectPlan (note "Document describing how project is to be conducted, organized, etc.") with a single value of type Document,
+	described by projectStart (note "Start date of project.") with a single value of type dateTime,
+	described by projectEnd (note "End date of project.") with a single value of type dateTime,
+	described by projectStandards (note "Set of standards used in project.") with values of type string,
+	described by projectQualityControlPlan (note "Document describing the quality control plan for the the site performing the project.") with a single value of type Document,
+	described by projectQualityProcessControlPlan (note "Document describing the quality process control plan for the the site performing the project.") with a single value of type Document,
+	described by projectDeliveryDate (note "Requested delivery date of project.") with a single value of type dateTime,
+	described by projectDistributionControl (note "ITAR-like restrictions.") with a single value of type string,
+	described by projectDataset (note "Datasets generated from the project.") with values of type DatasetMetadata,
+	described by projectReport (note "Reports of the project.") with values of type Document,
+	described by projectDataManagementPlan (note "Data management plan of the project.") with a single value of type Document,
+	described by parts (note "Parts created in this project") with values of type Part.

```

Comment:
Remove this OP because "Part" is defined in the part file. Instead, define a Project OP in the Part class.

Reply:
I removed the parts object property from Project and added a project object property in Part.